### DISCUSSION TITLE
Generate "prisma-client-dart", support server-side Dart?

### DISCUSSION THREAD

**Discussion by danielmahon** [prisma/prisma#9424] [Q&A] [answered]
I've been using Dart server-side more and more lately, and I have a few NodeJS microservices that I would love to test converting. The AOT compiler in Dart has the enticement of making these services much simpler/faster/smaller than with NodeJS. Thoughts on adding Dart (not Flutter) support, similar to the Go client?

**Comment by ryands17** [comment]
@danielmahon 👋 
I think we already support `Dart` as I had seen someone open an issue related to `Dart` as they were working with it: https://github.com/prisma/prisma/issues/8428

**Reply by danielmahon** [reply]
I think that issue refers to using the json output in a dart/flutter client-side app. I am looking for a dart only (excluding engine) server-side generated client. No JavaScript/Typescript involved anywhere in the stack. Like the Go "client"... but for dart.

**Reply by ryands17** [reply]
Then I would suggest creating a new feature request for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

**Reply by danielmahon** [reply]
Created https://github.com/prisma/prisma/issues/9473

**Comment by danielmahon** [comment, marked as answer]
Closing this discussion in favor of opening a new feature request here: https://github.com/prisma/prisma/issues/9473

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
